### PR TITLE
partition related apis require corresponding permissions

### PR DESCRIPTION
**Pull request by PowderLi**
issue #212

### CODE REVIEW

File: proto/milvus.proto [hunk 1]
```diff
@@ -1115,6 +1145,11 @@ message CompactionMergeInfo {
 }
 
 message GetFlushStateRequest {
+  option (common.privilege_ext_obj) = {
+    object_type: Collection
+    object_privilege: PrivilegeFlush
+    object_name_indexs: 4
```
Comment (SimFG) [inline proto/milvus.proto:1151]: should be `object_name_index: 4`

File: proto/milvus.proto [hunk 2]
```diff
@@ -1127,6 +1162,11 @@ message GetFlushStateResponse {
 }
 
 message GetFlushAllStateRequest {
+  option (common.privilege_ext_obj) = {
+    object_type: Collection
+    object_privilege: PrivilegeFlush
+    object_name_indexs: -1
```
Comment (SimFG) [inline proto/milvus.proto:1168]: should be `object_name_index: -1`

Comment (czs007) [inline proto/milvus.proto:1168]: should be object_name_index?

File: proto/milvus.proto [hunk 3]
```diff
@@ -1127,6 +1162,11 @@ message GetFlushStateResponse {
 }
 
 message GetFlushAllStateRequest {
+  option (common.privilege_ext_obj) = {
```
Comment (czs007) [inline proto/milvus.proto:1165]: This time we won't expose FlushAll.

**Pull request merged**